src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -198,4 +198,17 @@
   <data name="ERROR_CreatingVcxRequest" xml:space="preserve">
     <value>[CFamily Analysis] Error creating VCX request. File: {0}, Error: {1}</value>
   </data>
+  <data name="MsgHandler_ReportInvalidInput" xml:space="preserve">
+    <value>[CFamily Analysis] Internal error calling the analysis subprocess: {0}</value>
+  </data>
+  <data name="MsgHandler_ReportUnexpectedFailure" xml:space="preserve">
+    <value>[CFamily Analysis] Internal error in the analysis subprocess: {0}</value>
+  </data>
+  <data name="MsgHandler_ReportUnsupportedConfiguration" xml:space="preserve">
+    <value>[CFamily Analysis] Unsupported configuration: {0}</value>
+  </data>
+  <data name="MSG_GenericAnalysisFailed" xml:space="preserve">
+    <value>Check the output window for more information.</value>
```

Comment:
This text is passed to the status notifier which displays it both in the status bar and the output window. The wording is ok for the status bar but it's a bit clumsy for the output window if that is where the user reads it. Hopefully clear enough. Feel free to suggest alternatives.

Reply:
The status notifier only needs the file name, the failure message that we pass is the one that is shown in the output window.
https://github.com/SonarSource/sonarlint-visualstudio/blob/2fcabc8baab30c3dde67dc359290d602932cb65d/src/Integration.Vsix/Analysis/AnalysisStatusNotifier.cs#L74

So this message will always be shown only in the output window. We can change it to something generic, "internal error" / "analysis error".

Reply:
You're right: this is what appears in the status bar:

![image](https://user-images.githubusercontent.com/10757559/133402489-f2522b36-14ff-402e-b863-cb9d44034700.png)

The wording is very similar to what I had in an earlier message, so I thought the status bar was displaying my "new" message.

I'll change it as discussed offline.